frontend/ws3dash.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "Central Florida Event Siting Tool Output"
+output:
+  flexdashboard::flex_dashboard
+---
+
+```{r setup, include=FALSE, message=FALSE, warning=FALSE}
+library(flexdashboard)
+library(sf)
+library(mapview)
+library(tidyverse)
+library(DT)
+library(here)
+```
+
+```{r setup-parameters}
+# Set up main mapping parameters here
+
+# how many points to display on the map
+MAP_SITES_TO_DISPLAY <- 100
+# how many rows to display on the table
+TABLE_SITES_TO_DISPLAY <- 25
+# how many decimals to round the event score to
+EVENT_SCORE_NUM_DECIMALS <- 2
+```
```

Comment:
Finally, I added most of the non-hardcodable parameters to this code chunk so it's easy for people to change in the future.